### PR TITLE
Add File Picker documentation for "viewType" configuration parameter

### PR DESCRIPTION
**Pull request by KevinTCoughlin**
This adds documentation for the `viewType` configuration parameter which will be supported by `v7.2` of the OneDrive File Picker (pending release).

The `viewType` parameter allows developers to scope the selection of item types to one of `"all" | "files" | "folders"` with the default value being `"files"` which is the existing _default_ behavior.

An example usage which scopes item selection to _only_ folders is:

```js
var options = {
    clientId: '<client_id>',
    viewType: 'folders',
    success: (files) => console.log('success', files)
};

OneDrive.open(options);
```

Related to enabling support for #352.

### CODE REVIEW

**Comment by daspek** [comment]
OPS Build status updates of commit _[af48e91](https://github.com/KevinTCoughlin/onedrive-api-docs/pull/970/commits/af48e91daa0b3a9442d5e18b84213f228239a79f)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 8 sec(s)_

**Comment by daspek** [comment]
OPS Build status updates of commit _[af48e91](https://github.com/KevinTCoughlin/onedrive-api-docs/pull/970/commits/af48e91daa0b3a9442d5e18b84213f228239a79f)_: 

:clock10: **Incremental building**: average incremental building time is _28 sec(s)_

**Comment by daspek** [comment]
OPS Build status updates of commit _[af48e91](https://github.com/KevinTCoughlin/onedrive-api-docs/pull/970/commits/af48e91daa0b3a9442d5e18b84213f228239a79f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/controls/file-pickers/js-v72/open-file.md](https://github.com/KevinTCoughlin/onedrive-api-docs/blob/file-picker-viewtype/docs/controls/file-pickers/js-v72/open-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/controls/file-pickers/js-v72/open-file?view=odsp-graph-online&branch=pr-en-us-970) (>= odsp-graph-online) |
[docs/controls/file-pickers/js-v72/save-file.md](https://github.com/KevinTCoughlin/onedrive-api-docs/blob/file-picker-viewtype/docs/controls/file-pickers/js-v72/save-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/controls/file-pickers/js-v72/save-file?view=odsp-graph-online&branch=pr-en-us-970) (>= odsp-graph-online) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C3%5C1b9326b4-eb89-99a2-12be-09cf82b65094%5CPullRequest%5C201901030006038102-970%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4eLAAvK1IWC6mFcz5eLkmCCaYLDD751tlni5fjujVRQ%3D&st=2019-01-03T00%3A02%3A14Z&se=2019-02-03T00%3A07%3A14Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/controls/file-pickers/js-v72/open-file.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ var odOptions = {
 | **clientId**        | The application ID generated by the app registration console for your integration.                                                                                                                                                                                                            |
 | **action**          | The action type being performed with the files selected. You can specify `share` to generate a sharable URL, `download` to receive a short-lived URL that downloads the content of the files, or `query` to return identifiers that can be used with the Microsoft Graph API or OneDrive API. |
 | **multiSelect**     | The default value is **false**, which requires the user to select a single file only, or **true** to enable the user to select multiple files.                                                                                                                                                |
+| **viewType** | The type of item which can be selected. The default value is `files`. You can specify `folders` to limit selection to _only_ folders or specify `all` which enables the selection of _both_ files and folders. |
```
Comment (natalieethell) [inline docs/controls/file-pickers/js-v72/open-file.md]: Looks good! I'm not sure if it's just me, but I would probably write "The type of item **that** can be selected."

Comment (KevinTCoughlin) [reply docs/controls/file-pickers/js-v72/open-file.md]: I think you're correct here 😃 👍

**Comment by daspek** [comment]
OPS Build status updates of commit _[90df9e2](https://github.com/KevinTCoughlin/onedrive-api-docs/pull/970/commits/90df9e25ba0e1383eb64edb69f40472fcb4a95de)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 5 sec(s)_

**Comment by daspek** [comment]
OPS Build status updates of commit _[90df9e2](https://github.com/KevinTCoughlin/onedrive-api-docs/pull/970/commits/90df9e25ba0e1383eb64edb69f40472fcb4a95de)_: 

:clock10: **Full build**: average full build time is _49 sec(s)_ [calculated from last _1_ build(s)]

> Note: A full build takes more time than an incremental build.  This is the expected behavior of the build system.

**Comment by daspek** [comment]
OPS Build status updates of commit _[90df9e2](https://github.com/KevinTCoughlin/onedrive-api-docs/pull/970/commits/90df9e25ba0e1383eb64edb69f40472fcb4a95de)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/controls/file-pickers/js-v72/open-file.md](https://github.com/KevinTCoughlin/onedrive-api-docs/blob/file-picker-viewtype/docs/controls/file-pickers/js-v72/open-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/controls/file-pickers/js-v72/open-file?view=odsp-graph-online&branch=pr-en-us-970) (>= odsp-graph-online) |
[docs/controls/file-pickers/js-v72/save-file.md](https://github.com/KevinTCoughlin/onedrive-api-docs/blob/file-picker-viewtype/docs/controls/file-pickers/js-v72/save-file.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/controls/file-pickers/js-v72/save-file?view=odsp-graph-online&branch=pr-en-us-970) (>= odsp-graph-online) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C3%5C1b9326b4-eb89-99a2-12be-09cf82b65094%5CPullRequest%5C201901032107478009-970%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=xXZ0SxfNxamdka3gqalsY%2FLemcouMTVOHYEZljaqGvI%3D&st=2019-01-03T21%3A05%3A26Z&se=2019-02-03T21%3A10%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**